code/game/objects/items/storage/smartpack.dm

```diff
@@ -263,7 +265,8 @@
 
 
 /obj/item/storage/backpack/marine/smartpack/proc/repair_form(mob/user)
-	if(!ishuman(user) || activated_form || repairing)
+	if(!istype(user) || activated_form || repairing || user.stat == DEAD)
```

Comment:
You ought to either to type out the user var more, or restore the old ishuman check here.

Reply:
The others are istype, i thought it would cause an issue if it was inshuman since synths arent considered humans.
But i will change it back to ishuman, maybe i am overthinking it!

Reply:
Just for clarity the istype check here since `repair_form(mob/user)` user is typed to be a `/mob` it will ensure that user is a mob type. Whereas for `protective_form(mob/living/carbon/human/user)` user it typed to be a `/mob/living/carbon/human` so an istype check will ensure it is a human.